scripts/variantstore/wdl/CreateCohortTable.wdl

```diff
@@ -2,8 +2,17 @@ version 1.0
 
 workflow CreateCohortTable {
    input {
-        String project
+        String data_project
         String dataset
```

Comment:
maybe default_dataset for consistency?  wdyt?

Reply:
nice, yeah, will change